base/tps/shared/conf/db.ldif

```diff
@@ -41,6 +41,12 @@ objectClass: groupOfUniqueNames
 cn: Administrators
 description: Administrators for TPS
 
+dn: cn=TPS Auditors,ou=Groups,{rootSuffix}
+objectClass: top
+objectClass: groupOfUniqueNames
+cn: TPS Auditors
+description: Auditors for TPS
+
```

Comment:
As you saw in acl.ldif, the group name probably needs to be "Auditors" instead of "TPS Auditors".